src/sdkRuntimeModels.ts

```diff
@@ -34,7 +34,10 @@ export interface SDKEvent {
     IsFirstRun: boolean;
     EventName: string;
     EventCategory: number;
+
+    // https://go.mparticle.com/work/SQDSDKS-5196
     UserAttributes?: { [key: string]: string | string[] | null };
+
```

Comment:
is this whitespace intentional?

Reply:
Yes. It's to show that the comment only applies to that specific type definition.